cli/tests/integration_tests.rs

```diff
@@ -80,9 +80,13 @@ fn js_unit_tests() {
   assert!(status.success());
 }
 
+#[path = ""]
```

Comment:
Not ideal, but it works...

Also, sorry about the build failures... platform specific issues.

Reply:
Please add a comment here